packages/indicators/src/types.ts

```diff
@@ -15,6 +15,17 @@ export interface Analytic extends AnalyticValue {
   readonly dataElement: string;
 }
 
+export type DataValues = Record<string, DbValue>;
```

Comment:
Not sure if events values are always `string` or `string|number`?

```suggestion
export type DataValues = Record<string, string>;
```

Reply:
I think there's number as well because we are now sanitizing internal binary data to 1/0